Segment-Intercom/Classes/SEGIntercomIntegration.m

```diff
@@ -21,14 +21,11 @@ - (instancetype)initWithSettings:(NSDictionary *)settings
         self.settings = settings;
         self.intercom = [Intercom class];
         
-        NSString *apiKey = settings[@"apiKey"];
-        NSString *iOSAppId = settings[@"iOSAppId"];
+        NSString *mobileApiKey = settings[@"mobileApiKey"];
+        NSString *appId = settings[@"appId"];
```

Comment:
Confirming this change is actually intended?

Reply:
As in, did server side settings change?

Reply:
I had to change this because we can't expose the `apiKey` setting for server-side